docs/other/install-cvmfs-stratum1.md

```diff
@@ -0,0 +1,338 @@
+
+Install a CVMFS Stratum 1
+===================================================================
+
+# About this Document 
+This document describes how to install a CVMFS Stratum 1. There are many different variations on how to do that, but this document focuses on the configuration of the OSG GOC Stratum 1 oasis-replica.opensciencegrid.org. It is applicable to other Stratum 1s as well, very likely with modifications (some of which are suggested in the document below).
+
+# Applicable versions 
+The applicable software versions for this document are cvmfs and cvmfs-server >= 2.2.1.
+
+# Requirements
+
+## Host and OS
+
+-   OS is a 64-bit <span class="twiki-macro SUPPORTED_OS"></span>.
+-   Root access
+-   SELinux disabled
```

Comment:
Is this really necessary anymore?

Reply:
Not sure, Dave Dykstra will review this next week

Reply:
Ok, I was just curious so don't let it hold up the doc conversion since it's in the old doc anyway.

Reply:
Also, could you add Dave as a reviewer to the pull?  I can't do that since I don't have sufficient privileges

Reply:
Can't since he's not an owner of the repo. We don't have a formal review process for the docs yet so he can just review it once it's merged.